### PR TITLE
Add blog post for dao launch

### PR DESCRIPTION
**Pull request by m52go**

### CODE REVIEW

File: _posts/2019-04-17-dao-launch-third-anniversary.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The Bisq DAO has been in the works since work on Bisq begun back in 2014, but it's been iterated conceptually and technically many times since then to ensure it can achieve truly decentralized management and funding of the Bisq network.
+
+[The constraints](https://twitter.com/bisq_network/status/1103293547711746050) were not trivial: there could be no centrally-owned wallets, no legal entities, no investors, etc.
+
+Governance is always a tough thing to tackle, especially for a non-protocol project where people are needed to consistently deliver development, support, and other services, but the Bisq DAO seems to offer a promising solution. It offers a system of incentives that pushes all stakeholders to make the project sustainable simply by acting in their own best interest. The resulting dynamic challenges many social, economic, and organizational dynamics in significant ways, which we've started to cover in [this series](https://bisq.network/blog/dao-benefits-funding/).
+
+Early indications are encouraging: about 150 trades used BSQ to pay trading fees in the first 24 hours it was live.
```
Comment (ManfredKarrer) [inline _posts/2019-04-17-dao-launch-third-anniversary.md]: 150? Where do u have that number? I see 64 trades so far in the trades stats

File: _posts/2019-04-17-dao-launch-third-anniversary.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Many people from Bisq's past and present joined the call live on Zoom and on the YouTube livestream. It was a lively event—feel free to check it out:
+
+<div class='responsive-youtube-container'>
+    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/yCmpAqOCUSA?start=540" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+
+That was Monday.
+
+On Friday, Bisq celebrates 3 years in production. Volume trends indicate bitcoin-fiat volume in core markets is stronger than ever.
```
Comment (ManfredKarrer) [inline _posts/2019-04-17-dao-launch-third-anniversary.md]: Monero hit another peak yesterday , about 150 btc!

**Review by ManfredKarrer — APPROVED** [review]
Beside the comments, utACK

File: _posts/2019-04-17-dao-launch-third-anniversary.md (hunk 1 above)
Comment (m52go) [reply _posts/2019-04-17-dao-launch-third-anniversary.md]: dao > facts & figures > bsq transactions > number of fee payments transactions shows 207 right now, and it was about 150 late yesterday morning.

i could be wrong...am i understanding that number correctly?

Comment (m52go) [reply _posts/2019-04-17-dao-launch-third-anniversary.md]: actually 129 is the exact number, just counted trading fee transactions on the bsq block explorer from genesis up to 11am april 16.

Comment (ghost) [reply _posts/2019-04-17-dao-launch-third-anniversary.md]: There are 65 BSQ/BTC trades atm.
But I think that m52go speaks about trades using BSQ to pay the fees. And that is indeed nice.

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Comment by ManfredKarrer** [comment]
Leave merge to yourself...

**Pull request merged**